.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -0,0 +1,97 @@
+---
+name: Server migration
+about: Checklist to migrate to an upgraded server
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Checklist based on general guide https://github.com/openfoodfoundation/ofn-install/wiki/Migrating-a-Production-Server
+
+## 1. Setting up the new server
+- [ ] Check old server config for any additional services to be aware of (eg `/etc/nginx/sites-available`). Document any necessary steps for migration.
+- [ ] Hosting: provision new server with Ubuntu 20
+- [ ] DNS: add temporary domain (eg `prod2.openfoodnetwork.org`)
+
+### config
+- [ ] Add temporary name to `inventory/hosts`
+- [ ] Review `host_vars/x/config.yml`, clean up if needed
+  - Make a copy for the temp hostname, add temp domain to bottom of `certbot_domains`
+- [ ] Review `ofn-secrets:x_prod/secrets.yml`, clean up if needed
+   - Change to shared bugsnag projects
+   - Don't bother making a copy of this one
+
+### setup
+Ensure you have the correct secrets.
+`ansible-playbook -l x_prod2 -e "@../ofn-secrets/x_prod/secrets.yml playbooks/`
+- [ ] `letsencrypt_proxy.yml`
```

Comment:
Isn't this one run on the old server?

```
ansible_playbook playbooks/letsencrypt_proxy --limit current-prod -e "proxy_target=new-prod.org"
```

Reply:
Good pickup thanks, I must have mixed it up when formatting the list.